unit-tests/src/test/scala/scala/scalanative/regex/PatternTest.scala

```diff
@@ -213,69 +221,73 @@ object PatternSuite extends tests.Suite {
     fail("a|b", "c")
   }
 
-  test("quotation") {
+  @Test def quotation(): Unit = {
     pass("\\Qa|b|c\\E", "a|b|c")
   }
 
-  test("named groups") {
+  @Test def namedGroups(): Unit = {
     pass("(?<foo>a)", "a")
   }
 
-  test("non-capturing groups") {
+  @Test def nonCapturingGroups(): Unit = {
     pass("(?:a)", "a")
   }
 
-  test("flags") {
+  @Test def flags(): Unit = {
     pass("(?i)iNsEnSiTiVe", "insensitive")
     pass("(?i:iNsEnSiTiVe)", "insensitive")
   }
 
-  test("toString") {
+  @Test def testToString(): Unit = {
     val in = "foo|bar"
-    assert(Pattern.compile(in).toString == in)
+    assertTrue(Pattern.compile(in).toString == in)
   }
 
-  ignore("(Not supported) character classes (union and intersection)") {
+  @Ignore("not supported")
+  @Test def characterClassesUnionAndIntersection(): Unit = {
     pass("[a-d[m-p]]", "acn")
     pass("[[a-z] && [b-y] && [c-x]]", "g")
     pass("[a-z&&[def]]", "e")
     pass("[a-z&&[^aeiou]]", "c")
     fail("[a-z&&[^aeiou]]", "o")
     pass("[a-z&&[^m-p]]", "c")
     fail("[a-z&&[^m-p]]", "n")
-    pending // 620
+    //pending() // 620 (pending not supported in JUnit)
```

Comment:
That was not really supported in `Suite`, since it was just a no-op method. I believe `@Ignore` fills the role. I would suggest to remove those lines. Nobody is going to remember what those `pending` were supposed to be anyway.

Reply:
Pending just returned `()` but I was unsure what `// 620` meant. I will remove.

Reply:
Changes pushed.